### ISSUE TITLE
Legal process of registration of the foundation

### ISSUE THREAD

**Issue by Etiene** [lua-users-foundation/foundation#4]
Let's define what are the things we need for that?

We need volunteers to research into the legal process of registering a nonprofit organisation in different countries, so we make a decision about where to register and make up a list of what will be the next steps for registering it.

Last year, we were inclined to registering it in UK, so we need a volunteer to research about the process there. 

Given that, if anyone else has experience with non profits in other countries and is willing to volunteer for this task, please share your thoughts with us.

**Comment by RyanSquared** [comment]
I live in Kansas, which isn't the most desirable location, but could help with setting up a US-based 501(c)(3) nonprofit organization if the need arises.

**Comment by Etiene** [comment]
what are the requirements for registering a 501(c)(3)?

**Comment by catwell** [comment]
A reason why we probably don't want a US-based entity is that we intend to get funding from Russian companies, and it could make things difficult.

That being said, a 501(c)(3) is the standard. Both the Python Software Foundation and the Mozilla Foundation have that status.

**Comment by RyanSquared** [comment]
To quote IRS publication 4220 R.7-2014:

---

There are three key components for an organization to be exempt from federal
income tax under section 501(c)(3) of the IRC. A not-for-profit (i.e., nonprofit)
organization must be organized and operated exclusively for one or more exempt
purposes.

Organized – A 501(c)(3) organization must be organized as a corporation, trust,
or unincorporated association. An organization’s organizing documents (articles
of incorporation, trust documents, articles of association) must:
- limit its purposes to those described in section 501(c)(3) of the IRC;
- not expressly permit activities that do not further its exempt purpose(s), i.e.,
unrelated activities; and,
- permanently dedicate its assets to exempt purposes.

Operated – Because a substantial portion of an organization’s activities must
further its exempt purpose(s), certain other activities are prohibited or restricted
including, but not limited to, the following activities. A 501(c)(3) organization:
- must absolutely refrain from participating in the political campaigns of candidates
for local, state, or federal office;
- must restrict its lobbying activities to an insubstantial part of its total activities;
- must ensure that its earnings do not inure to the benefit of any private
shareholder or individual;
- must not operate for the benefit of private interests such as those of its founder,
the founder’s family, its shareholders or persons controlled by such interests;
- must not operate for the primary purpose of conducting a trade or business that
is not related to its exempt purpose, such as a school’s operation of a factory; and,
- must not have purposes or activities that are illegal or violate fundamental
public policy.

Exempt Purpose – To be tax exempt, an organization must have one or more
exempt purposes, stated in its organizing document. Section 501(c)(3) of the IRC
lists the following exempt purposes: charitable, educational, religious, scientific,
literary, fostering national or international sports competition, preventing cruelty
to children or animals, and testing for public safety.

---

In short, funds donated to the organization should be used for the benefit of the
organization as a whole, towards the policies established int he organization that
meet the requirements of the 501(c)(3), which includes scientific research, which
the Lua Users Foundation could be classified as.

**Comment by Etiene** [comment]
Brilliant!

Could anyone volunteer in researching documents that need to be provided, fees to pay, etc. for registering non-profit foundations in the US and in the UK?
